AdyenActions/UI/View Controllers/DelegatedAuthentication/DAErrorViewController.swift

```diff
@@ -81,24 +81,92 @@ internal final class DAErrorViewController: UIViewController {
                 return localizedString(.threeds2DADeletionConfirmationButtonTitle, localizationParameters)
             }
         }
+                
+        internal var troubleshootingSection: (
+            content: (title: String, description: String),
+            button: String
+        )? {
+            switch self {
+            case let .authenticationFailed(localizationParameters):
+                return (
+                    content: (
+                        title: localizedString(.threeds2DAErrorTroubleshootingTitle, localizationParameters),
+                        description: localizedString(.threeds2DAErrorTroubleshootingDescription, localizationParameters)
+                    ),
+                    button: localizedString(.threeds2DAErrorTroubleshootingButtonTitle, localizationParameters)
+                )
+            case .registrationFailed, .deletionConfirmation:
+                return nil
+            }
+        }
+        
+        internal var resetCredentialAlert: (
```

Comment:
Looking at these tuples (this and `troubleshootingSection`, could it be better to create a nested struct here with 4 properties (`negativeButtonTitle` as optional) and make more readable?
`contentTitle`, `contentDescription` and so on

Reply:
My thoughts here. I did think of creating a struct here, but thought that a tuple is more suited as it 
1. avoids creating another type where usage is limited to one place & file lenght.
2. I expected that this would be better readable & easily usable as we have only non optional types. (i find named tuples readable similar to struct but that may be my personal bias).

Regarding creating one struct for both an `alert` and the `troubleShootingSection` 
If i go this route, i would still prefer to create a struct for each of them like `AlertDetails` &`TroubleShootingSectionDetails` as 
1. the purpose of the type is clear. 
2. If there is a design update to the troubleshooting section i wouldn't need to update the shared type.

What would i name the shared struct if it is one type?
Also an alert would have a postiive and negative button, but not the troubleshooting addition to the alert screen. 

I'm still leaning on keeping it as-is and using the tuples. But pls let me know if you guys think that a struct is better i'll happily switch.

Reply:
I see, fair points.